packages/lit-dev-content/samples/examples/react-basics/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "/samples/base.json",
+  "order": 0,
+  "title": "Basics",
+  "description": "Render web components in React apps.",
+  "section": "React",
+  "files": {
+    "app.tsx": {"contentType": "text/typescript-jsx"},
```

Comment:
@e111077 even though this file is listed first, when navigating from another playground with `simple-greeting.ts` in it and focused, `simple-greeting.ts` will be selected instead of `app.tsx`. In this case, without prose to guide the user, I think it's really bad to start them there and not in some react code. Could we fix that? @taylor-vann maybe in this PR if it's simple enough?

@taylor-vann could you add a comment to this component stating that this is just a demo web component that we want to show used in React and that to check out `app.tsx` to see the call to `createComponent()`? And maybe change the name of the component if we can't fix the playground issue quickly.

Reply:
Sure thing! Lemme see what i can do

Reply:
renamed simple-greeting to demo-greeting